mhkit/wave/io/hindcast/wind_toolkit.py

```diff
@@ -417,7 +417,12 @@ def request_wtk_point_data(
     hash_params = f"{time_interval}_{parameter}_{lat_lon}_{years}_{preferred_region}_{tree}_{unscale}_{str_decode}_{hsds}"
 
     # Use handle_caching to manage caching.
-    data, meta, _ = handle_caching(hash_params, cache_dir, clear_cache_file=clear_cache)
+    data, meta, _ = handle_caching(
+        hash_params,
+        cache_dir,
+        cache_content={"data": None, "metadata": None, "write_json": None},
+        clear_cache_file=clear_cache,
+    )
```

Comment:
handle_caching reduced to only 5 inputs
https://pylint.readthedocs.io/en/stable/user_guide/configuration/all-options.html#max-args